{{cookiecutter.project_name}}/dev-docker-compose.yml.default

```diff
@@ -0,0 +1,42 @@
+# Configuration for local development
+#
+# See README for setup and usage instructions
+#
+version: '3'
+
+services:
+  postgres:
+    image: postgres
```

Comment:
Not sure if we should be pinning here? may be best to leave that to a per project basis or have it use an env var that defaults to latest

Reply:
I'd go with no pinning. It's only a starting point.

Reply:
Using an env to use latest by default, which can be overridden.